docs/source/gateway/index.rst

```diff
@@ -634,15 +636,20 @@ If a given provider does not provide support for a parameter, a clear message wi
 
 - **candidate_count** (Supported by OpenAI, Cohere): This parameter determines the number of alternative responses to generate. The range is from 1 to 5, and by default, it is set to 1.
 
-Alternate Query Parameters
---------------------------
-There are additional provider-specific parameters that will work (i.e., ``logit_bias`` (OpenAI, Cohere), ``frequency_penalty`` (OpenAI, Cohere), ``presence_penalty`` (OpenAI, Cohere), and ``top_k`` (Anthropic, Cohere)) with the exception of the following:
```

Comment:
This wording makes it sound like these parameters are specified *instead* of the standard parameters (the word "alternate" is inaccurate) and also makes it sound like the listed parameters are the only ones supported ("i.e" is used instead of something like "e.g").

Fixed here.